presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestAllDatatypesFromHiveConnector.java

```diff
@@ -275,6 +345,46 @@ else if (usingTeradataJdbcDriver(connection)) {
         }
     }
 
+    private void avroAssertColumnTypes(QueryResult queryResult)
```

Comment:
please inline this method. It matches only `QueryResult` for particular query so it cannot be reused.